### ISSUE TITLE
?: key value notation doesn't work.

### ISSUE THREAD

**Issue by rcls** [k14s/ytt#57]
#! Trying to use the multiline yaml ? : notation for map entries doesn't give the expected results:

#! This works
? ac
: bd

#! But this does not (it gives ee:null not ee:fg)
? ee
: #@ 'f' + 'g'

#! Nor this (it gives null:ax not ax:bd)
? #@ 'a' + 'x'
: #@ 'b' + 'd'

#! This one gives an error if you uncomment it.
#! ? #@ 'y' + 'z'
#! : asdf

**Comment by cppforlife** [comment]
related to https://github.com/k14s/ytt/issues/29

**Comment by cppforlife** [comment]
@rcls from #29 you were trying to use this syntax for templated string keys. are you still doing that or now it's non-string case?

**Comment by rcls** [comment]
> @rcls from #29 you were trying to use this syntax for templated string keys. are you still doing that or now it's non-string case?

I came across this because it's seems convenient for use with templated string keys.

(I'm not crazy enough to use maps or sequences as keys. :-) )

**Comment by cppforlife** [comment]
does `@yaml/map-key-override` worked out for you from #29?

**Comment by rcls** [comment]
> did `@yaml/map-key-override` worked out for you from #29?

Yes, the change done for #29 is working beautifully (& has given me a major clean-up of some fairly complex templates).

The current request for ?: support is very much an extra nice-to-have.
